frontend/src/views/SettingsView.vue

```diff
@@ -196,13 +203,11 @@ export default {
     /**
      * Get the status description for a status code
      */
-    getStatusDescription(statusCode) {
-      switch (statusCode) {
-        case CRM_STATE_CODES.ACTIVE:
-          return 'Active'
-        case CRM_STATE_CODES.INACTIVE:
-          return 'Inactive'
+    getStatusDescription(user) {
+      if (this.isDeactivatedUser(user) || user?.stateCode === CRM_STATE_CODES.INACTIVE) {
```

Comment:
Based on discussions we may treat Inactive users as soft deleted in the future meaning we won't even see them in the portal. But this is fine for now.

Reply:
agreed!